src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -230,6 +230,12 @@ Plugin::Plugin()
           [](const Config& config) {
               return config.get<PERFORMANCE_HINT>();
           }}},
+        {ov::hint::execution_mode.name(),
+         {true,
```

Comment:
Noticed that you added a method `isPublic` for this property. Can we just use it instead of this `true`?

Reply:
I'd rather stick to the unitary solution, for better readability. If we decide to use isPublic instead (however it does not apply for all props), i'd rather do it in a separate PR for all props at once, for consistency